src/app/Home/book/show-books/show-books.component.css

```
@@ -0,0 +1,73 @@
+/* show-books.component.css */
+
+.container {
+    display: flex;
+}
+
+.book-list {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.book-row {
+    display: flex;
+    align-items: flex-start;
+    margin: 10px;
+    padding: 15px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
```

```suggestion
    margin: 0.62rem 0;
    padding: 0.94rem;
```
When we write like **margin: 10px;** (or **padding:0.94rem**) this, it actually creates margin (or padding) to top, left, bottom and right respectively. If you don't need any margin to left and right side we can write like as suggested. If you need it then you can keep as it is now.

**ditto**
please convert all the **px** to **rem** unit.